Kernel/Modules/AgentTicketZoom.pm

```diff
@@ -1253,7 +1254,7 @@ sub MaskAgentZoom {
                     },
                 );
 
-                for my $SubItem ( sort keys %{ $ZoomMenuItems{$Item}->{Items} } ) {
+                for my $SubItem ( sort { ($ZoomMenuItems{$Item}->{Items}->{$a}->{Priority} // 999) <=> ($ZoomMenuItems{$Item}->{Items}->{$b}->{Priority} // 999) } keys %{ $ZoomMenuItems{$Item}->{Items} } ) {
```

Comment:
@itweserems 

First of all, thank you for the PR 👍 

However, the sorting here does not seem to have the desired effect. Previously it was sorted by the keys of the hash, which look like e.g. `420-Customer`. Now it is sorted by the key `Priority` of the items hash, which in most cases is undefined/not present and in other cases contains a ticket priority like '3 normal'.

Can you please check?

Regards,
   Jens

Reply:
@itweserems @jepf 
As there was no further response, I will create an internal MR for QA.
We will then implement any further changes ourselves.